src/olympia/promoted/tests/test_utils.py

```diff
@@ -149,3 +150,23 @@ def test_create_stripe_checkout_session_with_custom_rate():
             ],
             customer_email=customer_email,
         )
+
+
+def test_create_stripe_customer_portal():
+    addon = addon_factory()
+    customer_id = "some-customer-id"
+    fake_portal = "fake-return-value"
+
+    with mock.patch(
+        "olympia.promoted.utils.stripe.billing_portal.Session.create"
+    ) as create_portal_mock:
+        create_portal_mock.return_value = fake_portal
```

Comment:
You define a return value here, but never check in the test that it's actually returned.

Reply:
good call